src/CitationProvider.ts

```diff
@@ -14,36 +14,40 @@
  * limitations under the License.
  */
 
-import locales from '@manuscripts/data/dist/csl/locales/locales.json'
 import { BibliographyItem } from '@manuscripts/json-schema'
 import CiteProc, { Citation as CiteProcCitation } from 'citeproc'
 
 import { variableWrapper } from './citeproc-variable-wrapper'
 import { convertBibliographyItemToCSL } from './csl-converter'
+import defaultLocale from './defaultLocale'
 
 interface Props {
   getLibraryItem: (id: string) => BibliographyItem | undefined
   citationStyle: string
-  lang?: string // Default en-GB
+  lang?: string // Default en-US
+  locale: string | null
```

Comment:
Would `locale?: string` work here?

Reply:
will I use null as axios request will return `ResponseType | null` so to keep it unified with it